dotcom-rendering/src/web/components/Card/Card.tsx

```diff
@@ -230,7 +230,11 @@ export const Card = ({
 							<div>
 								{standfirst && (
 									<StandfirstWrapper palette={cardPalette}>
-										{standfirst}
+										<div
+											dangerouslySetInnerHTML={{
+												__html: standfirst,
+											}}
```

Comment:
Will this receive the correct styling in terms of font, size, etc, or do we need an emotion global to take care of this like we have on articles?

Reply:
Looking at today's front, there is a single example of the `trailText` containing html, and it's a `strong` tag.
<img width="1235" alt="Screenshot 2022-04-22 at 08 40 36" src="https://user-images.githubusercontent.com/1336821/164634244-3aac0cd4-682f-4992-a2c3-aa0692fe34fd.png">
Looking in the Fronts Tool, it seems like this field is just plain text so I think there's a case to say this field is unlikely to contain html as you'd need to manually write it but it's not impossible.

That said, I recall the reason I decided to dangerously set it was because I had seen a p tag in use here and because I also see one other example I am inclined to keep this approach. But I don't think we need to add any additional styling. We can review this as more examples arrive.

Reply:
It's quite simple to add some logs for these cases, so we get a sense of how frequently they happen.